src/en/tut-lxd.md

```diff
@@ -0,0 +1,264 @@
+Title: Getting started with Juju and LXD
+
+# Getting started with Juju and LXD
+
+[LXD](lxd-upstream) provides an amazing experience using machine containers to be able to
```

Comment:
Reworded, thanks for suggestion. I like the phrase "cloud-like experience" in particular.